database/sql/V021__store_full_quotes.sql

```diff
@@ -0,0 +1,27 @@
+-- This migration step will change how we store min-fee measurements to store
+-- full quote data that can later be attached to an order.
+--
+-- This involves adding new columns for buy and sell amounts in the quotes as
+-- well as renaming the tables so its more apparent what they are for.
+-- Additionally, for the quotes table, we add a `bigserial` ID column for
+-- generating unique quote IDs for each order.
+
+ALTER TABLE min_fee_measurements RENAME TO quotes;
+ALTER INDEX min_fee_measurements_token_expiration RENAME TO quotes_token_expiration;
+DELETE FROM quotes;
+ALTER TABLE quotes
+    DROP COLUMN amount,
+    ALTER COLUMN buy_token SET NOT NULL,
+    ALTER COLUMN order_kind SET NOT NULL,
```

Comment:
For consistency with the other columns in this table. Not strictly needed by this PR though.